flood_api/dependencies/queryparams.py

```diff
@@ -1,23 +1,56 @@
 from datetime import date
-from typing import Annotated
+from typing import Annotated, Optional, Union
 
-from fastapi import Depends, Query
+from fastapi import Depends, HTTPException, Query
 
+from flood_api.settings import settings
+from flood_api.utils.validation_helpers import (
+    validate_bounding_box,
+    validate_coordinates,
+    validate_dates,
+)
 
-def coordinates(
-    lon: Annotated[
-        float,
-        Query(description="Longitude of the coordinate to retrieve data for"),
-    ],
-    lat: Annotated[
-        float,
-        Query(description="Latitude of the coordinate to retrieve data for"),
-    ],
-) -> (float, float):
-    return lon, lat
+INVALID_STATUS_CODE = settings.invalid_query_status_code
```

Comment:
Is this something we will ever want to configure? I think an invalid request should always be a 400. I don't think we need to store 400 in a constant anywhere